Sources/Tuplet.swift

```diff
@@ -387,6 +378,20 @@ public struct Tuplet: NoteCollection {
         try tuplet.insert(noteCollections, at: sliced)
         notes[flatIndex[0]] = tuplet
     }
+    
+    private mutating func removeNotes(at flatIndexes: [[Int]]) throws {
+        let counts = Set(flatIndexes.map { $0.count })
+        let flatIndexCountGroups = counts.map { count in
+            flatIndexes.filter { flatIndex in
+                return flatIndex.count == count
+            }
+        }.sorted(by: { $0[0][0] > $1[0][0] })
```

Comment:
Same as below. Do you need to sort them or are you just needing to reverse them? If the latter, you can just call `reversed()`, right?

Reply:
Same as the previous one